apps/modernization-ui/src/apps/page-builder/page/management/header/tabs/page-tabs.module.scss

```diff
@@ -33,4 +33,18 @@
             outline: 0.25rem solid #2491ff;
         }
     }
+    button {
+        display: flex;
+        align-items: center;
+        gap: 0.5rem;
+        margin-top: -0.75rem;
+    }
+    .modalBody {
+        background-color: pink;
```

Comment:
Issue: we should leverage colors in the `styles/_colors`. If an appropriate color is not in the stylesheet already, the design team should be consulted to determine if the figma used an incorrect color by accident.